macros/parsers/parserRadioMultiAnswer.pl

```diff
@@ -638,6 +648,9 @@ sub generate_aria_label {
 # Produce the answer rule.
 sub ans_rule {
 	my ($self, $size, @options) = @_;
+
+	return '' if !$self->{showInStatic} && ($main::displayMode eq 'TeX' || $main::displayMode eq 'PTX');
```

Comment:
To ensure that the answer is recorded, this line will need to be moved to after the for loop that begins at line 661 below.  Then another copy of this line will need to be added at line 773 in the `begin_radio` method.  Also add `next if !$self->{showInStatic} && ($main::displayMode eq 'TeX' || $main::displayMode eq 'PTX');` after line 663 in the for loop below (the `named_ans_array_extension` and `named_ans_rule_extension` methods do not need to be called contrary to what I said before).

Reply:
Thanks for working that out. I made those changes and force pushed.